addons/spreadsheet/static/src/data_sources/odoo_views_data_source.js

```diff
@@ -41,9 +28,9 @@ export class OdooViewsDataSource extends LoadableDataSource {
         this._metaData = JSON.parse(JSON.stringify(params.metaData));
         /** @protected */
         this._initialSearchParams = JSON.parse(JSON.stringify(params.searchParams));
-        this._initialSearchParams.context = removeContextUserInfo(
-            this._initialSearchParams.context
-        );
+        if (this._orm && this._orm.user) {
```

Comment:
@LucasLefevre 
Hello! Following the JS team advice, this is the preferred form for the context cleanup function.

In this file, I protected `this._orm` and `this._orm.user` because I'm not sure it's always there, but maybe it is? Do you think the protection is unnecessary?

Reply:
`_orm` is supposed to be defined. For `this._orm.user`, I don't know. It's propably supposed to be an internal things of the orm service. The user service should be used directly here (and not though the orm).
But can be changed later

Reply:
Thanks for the reply, I checked and the userService is a dependency of the ormService and `this._orm.user` is a reference to the userService, so no need for the check it seems :)